.github/workflows/dm_binlog_999999.yaml

```diff
@@ -53,6 +53,9 @@ jobs:
           mv tidb-enterprise-tools-nightly-linux-amd64/bin/sync_diff_inspector bin/
           curl http://download.pingcap.org/tidb-nightly-linux-amd64.tar.gz | tar xz
           mv tidb-nightly-linux-amd64/bin/tidb-server bin/
+          curl -O https://dl.min.io/server/minio/release/linux-amd64/minio
+          chmod +x minio
```

Comment:
it may be better that we  download like https://github.com/PingCAP-QE/ci/blob/main/jenkins/pipelines/ci/ticdc/dm_ghpr_integration_test.groovy#L232

Reply:
I'm afraid that github could not access fileserver.pingcap.net

Reply:
https://github.com/pingcap/tiflow/runs/5418761561?check_suite_focus=true

Reply:
oh. i see. it can not access fileserver.pingcap.net.